deepchecks/tabular/checks/train_test_validation/train_test_feature_drift.py

```diff
@@ -231,12 +231,11 @@ def add_condition_drift_score_not_greater_than(self, max_allowed_categorical_sco
         Parameters
         ----------
         max_allowed_categorical_score: float , default: 0.2
-            the max threshold for the categorical variable drift score
+            The max threshold for the categorical variable drift score
         max_allowed_numeric_score: float ,  default: 0.1
-            the max threshold for the numeric variable drift score
-        number_of_top_features_to_consider: int , default: 5
-            the number of top features for which exceed the threshold will fail the
-            condition.
```

Comment:
What about deprecating "number_of_top_features_to_consider"? Where did we land on that?

Reply:
That it is unneeded for this :)